python/lsst/meas/algorithms/subtractBackground.py

```diff
@@ -116,121 +116,35 @@ class SubtractBackgroundConfig(pexConfig.Config):
     )
 
 
-## @addtogroup LSST_task_documentation
-## @{
-## @page SubtractBackgroundTask
-## @ref SubtractBackgroundTask_ "SubtractBackgroundTask"
-## @copybrief SubtractBackgroundTask
-## @}
-
 class SubtractBackgroundTask(pipeBase.Task):
-    r"""!Subtract the background from an exposure
-
-    @anchor SubtractBackgroundTask_
-
-    @section meas_algorithms_subtractBackground_Contents  Contents
-
-     - @ref meas_algorithms_subtractBackground_Purpose
-     - @ref meas_algorithms_subtractBackground_Initialize
-     - @ref meas_algorithms_subtractBackground_IO
-     - @ref meas_algorithms_subtractBackground_Config
-     - @ref meas_algorithms_subtractBackground_Metadata
-     - @ref meas_algorithms_subtractBackground_Debug
-     - @ref meas_algorithms_subtractBackground_Example
-
-    @section meas_algorithms_subtractBackground_Purpose  Description
-
-    Fit a model of the background of an exposure and subtract it.
-
-    @section meas_algorithms_subtractBackground_Initialize  Task initialisation
-
-    @copydoc \_\_init\_\_
-
-    @section meas_algorithms_subtractBackground_IO  Invoking the Task
-
-    Call `run` to fit the background and subtract it.
-
-    Call `fitBackground` to fit the background without subtracting it.
-
-    @section meas_algorithms_subtractBackground_Config  Configuration parameters
-
-    See @ref SubtractBackgroundConfig
-
-    @section meas_algorithms_subtractBackground_Metadata  Quantities set in exposure Metadata
-
-    The `run` method will optionally set the following items of exposure metadata;
-    the names may be overridden; the defaults are shown:
-    <dl>
-        <dt>BGMEAN <dd>mean value of background
-        <dt>BGVAR  <dd>standard deviation of background
-    </dl>
```

Comment:
I think this above note about metadata should go into an "In Depth" section in the Task Topic for this.

https://developer.lsst.io/stack/task-topic-type.html#in-depth-section